libsession-util/src/main/java/network/loki/messenger/libsession_util/util/GroupInfo.kt

```diff
@@ -14,6 +14,7 @@ sealed class GroupInfo {
         val invited: Boolean,
         val name: String,
         val destroyed: Boolean,
+        val joinedAt: Long, // In seconds
```

Comment:
Could we rename this to `joinedAtSecs` and then we can remove the comment and it's a bit clearer throughout?